ITEM.tsv

```diff
@@ -8523,261 +8523,261 @@ ITEM_20151102_008522	Unsent Letters
+ITEM_20151224_008661	Stone Bracelet
+ITEM_20151224_008662	The stone filled with magic can be used to create bracelets.
+ITEM_20151224_008663	Superior Star Bracelet
+ITEM_20151224_008664	Superior Gem Bracelet
+ITEM_20151224_008665	Kranto Bracelet
+ITEM_20151224_008666	 - Defeating plant enemy will activate plant collection effect{nl}  -  Herb items can be droped when the plant collection effect is stacked 15 times
+ITEM_20151224_008667	Elements Dance
+ITEM_20151224_008668	Created by mages to protect element magics, but fatal flaw couldn't be solved.
+ITEM_20151224_008669	 - Avoiding enemy attack will activate frenzy effect{nl}  - Increases physical attack by 235, movement speed by 10 and reduces physical defense by 88 for 30 seconds when the frenzy effect is stacked 8 times.
+ITEM_20151224_008670	Templar Costume
+ITEM_20151224_008671	Sage Costume
+ITEM_20151224_008672	Warlock Costume
+ITEM_20151224_008673	Kabbalist Costume
+ITEM_20151224_008674	Hair Costume 1
+ITEM_20151224_008675	Hair Costume 2
+ITEM_20151224_008676	Hair Costume 3
+ITEM_20151224_008677	Hairpin precisely finished by master craftsman. Right-click to equip.
+ITEM_20151224_008678	New Hat 01
+ITEM_20151224_008679	White Ribbon Decoration Boater
+ITEM_20151224_008680 	Red Ribbon Decoration Boater
+ITEM_20151224_008681	Winter Hat
+ITEM_20151224_008682	New Hat 05
+ITEM_20151224_008683	New Hat 06
+ITEM_20151224_008684	Boater
+ITEM_20151224_008685	New Hat 08
+ITEM_20151224_008686	Noble Hat
+ITEM_20151224_008687	New Hat 10
+ITEM_20151224_008688	New Hat 11
+ITEM_20151224_008689	New Hat 12
+ITEM_20151224_008690	New Hat 13
+ITEM_20151224_008691	New Hat 14
+ITEM_20151224_008692	New Hat 15
+ITEM_20151224_008693	Dignity Miter
+ITEM_20151224_008694	Miter brimming with dignity. It's easy to catch the attention of others.
+ITEM_20151224_008695	Prestige Miter
+ITEM_20151224_008696	Others will listen to you more.
+ITEM_20151224_008697	Imposing Miter
+ITEM_20151224_008698	Can create an overwhelming atmosphere by wearing this.
+ITEM_20151224_008699	New Hat 19
+ITEM_20151224_008700	New Hat 20
+ITEM_20151224_008701	New Hat 21
+ITEM_20151224_008702	New Hat 22
+ITEM_20151224_008703	Tribal Chief Head Decoration
+ITEM_20151224_008704	Feather decoration is common characteristic between the tribes.
+ITEM_20151224_008705	Canine Head Decoration
+ITEM_20151224_008706	It might be disturbing during meals.
+ITEM_20151224_008707	New Hat 25
+ITEM_20151224_008708	New Hat 26
+ITEM_20151224_008709	New Hat 27
+ITEM_20151224_008710	New Hat 28
+ITEM_20151224_008711	New Hat 29
+ITEM_20151224_008712	Ritual Coffin
+ITEM_20151224_008713	Forecast of warm wind to breeze over their house. A rainbow to appear over their shoulder at all times.
+ITEM_20151224_008714	Cornus Helmet
+ITEM_20151224_008715	You can pick and eat the fruits if you ever feel hungry.
+ITEM_20151224_008716	Red Beret
+ITEM_20151224_008717	Reb Beret respresents outstanding movements. This beret is for those users participated in G-STAR 2015.
```

Comment:
A red beret that represents outstanding movements. This beret is for users who attended G-STAR 2015.